.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 sudo: false
 
 jdk:
-  - oraclejdk8
+  - oraclejdk9
```

Comment:
All samples do compile with JDK 9.

If we want to keep the 8 here, we could also switch the JDK for samples that rely on JDK 9;

```
    - stage: junit5-vanilla-java9
      script: jdk_switcher use oraclejdk9 && cd $TRAVIS_BUILD_DIR/junit5-vanilla-java9 &&./build.jsh
```

Reply:
Switch JDK on-the-fly didn't work at the first try: https://travis-ci.org/junit-team/junit5-samples/jobs/276785150#L470

But it could be done. Somehow. I can live with running JDK 9 on Travis CI though.

Reply:
Looks better now: https://travis-ci.org/junit-team/junit5-samples/builds/276956865